app/docs/src/how-tos/aws-eks.md

```diff
@@ -0,0 +1,520 @@
+`nodegroup-all-protocol-all-ports` component.
+
+Connect the `Security Group ID` output socket of
+eks-cluster-sg`component to the`Source Traffic Security Group
+ID`input socket of this`nodegroup-all-protocol-all-ports` component.
+
+### Apply your Change Set
+
+![Apply Change Set](./aws-eks/apply.png)
+
+Press `Escape` or click anywhere on the canvas background to select the
+Workspace.
+
+Click the `Apply Change Set` button to:
+
+- Create all of the desired IAM Roles and Policies
+- Create all of the Security Groups
+- Create an EKS Fargate Profile
+- Create an EKS NodeGroup
+
+NOTE: When creating a nodegroup and/or a fargate profile, AWS requires the
+cluster to be `ACTIVE`. The create actions will fail with `warning` messages.
+You can retry these actions after a period of time.
+
+## Accessing the cluster
+
+By default, the user used to create the cluster will be the user to have access
+to it. It is not recommended to use access keys directly with System Initiative.
+The workspace uses assumeRolePolicy, so specific users will need granded access
+to the cluster.
+
+### Generating a KubeConfig
+
+Using the AWS CLI, run the following command:
+
+```
+aws eks update-kubeconfig --name demo-eks-cluster --region us-east-1
+```
+
+The result is something similar to:
+
+```
+aws eks update-kubeconfig --name demo-eks-cluster --region us-east-1
+Added new context arn:aws:eks:us-east-1:573113295293:cluster/demo-eks-cluster to /Users/stack72/.kube/config
+```
+
+Using [kubectl](https://kubernetes.io/docs/reference/kubectl/), you can check
+the access permissions to the cluster:
+
+```
+kubectl auth can-i get pods
+```
+
+The result may be something similar to:
+
+```
+E1112 00:25:56.469718   29444 memcache.go:265] couldn't get current server API group list: the server has asked for the client to provide credentials
+E1112 00:25:57.309084   29444 memcache.go:265] couldn't get current server API group list: the server has asked for the client to provide credentials
+Warning: the server doesn't have a resource type 'pods'
+
+error: You must be logged in to the server (Unauthorized)
+```
+
+This is because my local user isn't part of the Cluster access configuration
+
+Start in a change set named `EKS Cluster Access`.
+
+### Update EKS Cluster Access Config
+
+![Update Cluster Config](./aws-eks/update-cluster-config.png)
+
+Update the `demo-eks-cluster` `accessConfig.authenticationMode` to be
+`API_AND_CONFIG_MAP`
+
+Right click on the component and select `Update Cluster Access Config`.
+
+Note: This change needs to take effect in the cluster before you can add access
+policies. So you will need to retry the failed access entry actions.
+
+### Create Access Entry
+
+![Create Access Entry](./aws-eks/create-access-entry.png)
+
+Add an `Access Entry` to your `demo-eks-cluster` eks cluster frame.
+
+Set the component name to be `My User`.
+
+Set the `type` to be `STANDARD`.
+
+Set the `principalArn` to be `<enter your userArn>`.
```

Comment:
Add a note here saying that this is an example user/user which you want to grant access to the cluster + give an example of a user arn in the correct form to help them